### ISSUE TITLE
train FeedbackHourGlass to detect landmarks

### ISSUE THREAD

**Issue by JingzheLyp** [Maclory/Deep-Iterative-Collaboration#27]
hi, buddy,
I want to know the way you train your FeedbackHourGlass model, the file FB_HG_68_CelebA.pth you used as a pretrained landmark detect model for DIC or DIC-GAN. When I train this model alone, the loss is very large. Can you tell me how you train it? Thanks a a lot.

**Comment by Onepengpeng** [comment]
Hi,I meet the same problem.Have you solved the issue ? @JingzheLyp
